asp_interactive_configuration/ooasp/encodings/ooasp_guess.lp

```diff
@@ -11,18 +11,18 @@
   	ooasp_domain(C,new_object),
 	guess.
 
+#heuristic ooasp_isa_leaf(LEAFCLASS,new_object): 
```

Comment:
Please add a brief comment why this heuristic is used.

Reply:
I removed it from now, since it did not give the expected result, the idea was to try to get the least insitanciated objects possible